compute/admin_guide/agentless-scanning/onboard-accounts/onboard-gcp.adoc

```diff
@@ -460,6 +460,12 @@ gcloud iam service-accounts keys create ${TARGET_PROJECT_NAME}-service_account_k
 [ "${GOOGLE_CLOUD_SHELL}X" == "trueX" ] && cloudshell download ${TARGET_PROJECT_NAME}-service_account_key.json
 ----
 
+. If you encounter the `failed to create snapshot for instance ...` error or `Error 403: Required 'compute.disks.createSnapshot' permission for ...` error, use this https://cdn.twistlock.com/docs/attachments/example-hub-target-permissions.tf.json[Terraform script] to grant the needed permissions across the target and hub accounts.
```

Comment:
the new location is 

https://docs.paloaltonetworks.com/content/dam/techdocs/en_US/pdf/prisma/prisma-cloud/prerelease/example-hub-target-permissions.tf.json

Reply:
Done